### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by jaredgalanis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,15 +30,18 @@ To run the entire release pipeline, do the following:
 
 For now you are required to specify the version being released, and only minor versions increments are supported.
 
-### Running the entire pipeline
+### Running a specific project step-by-step (recommended)
+
+You can use the commands documented in [learning team release process](https://github.com/ember-learn/handbook/blob/master/ember-releases.md) to run this tool step-by-step.
+
+If you need to run the pipeline for one of the projects, you can specify a `--project` (or `-p`) option when invoking the tool.
```
Comment (mansona) [inline README.md:37]: This isn't true any more. The "projects" have been moved to the sub commands now so  this paragraph can be deleted

File: README.md [hunk 2]
```diff
@@ -30,15 +30,18 @@ To run the entire release pipeline, do the following:
...
+See `tool-new-release --help` for the full list of projects.
+
+### Running the entire pipeline (not recommended)
```
Comment (mansona) [inline README.md:40]: The ability to run the whole pipeline has been removed so it can be deleted from the docs 👍

File: src/projects/guides.rs [hunk 3]
```diff
@@ -15,7 +15,7 @@ pub fn run(dir: &Path, dry_run: bool) {
     let version = Version::parse(&stdout_result[1..stdout_result.len()].trim()).unwrap();
 
     if  !req.matches(&version) {
-        panic!("Guides can only be installed with node version 16 right now. you have {:?}", stdout_result)
+        panic!("Guides can only be installed with node version 14 right now. you have {:?}", stdout_result)
```
Comment (mansona) [inline src/projects/guides.rs:18]: a) nice catch 😂 
b) is this true any more? now that we've released the redesign we don't use sass and we probably don't have any restrictions on Node version any more 🤔

**Review by mansona — CHANGES_REQUESTED** [review]
mostly some stuff to delete and one question 👍

File: src/projects/guides.rs (hunk 3 above)
Comment (jaredgalanis) [reply src/projects/guides.rs:18]: I think you're right. It's probably not accurate. Maybe we bump to v14 and above for the moment.

**Review by mansona — APPROVED** [review]
LGTM

**Pull request merged**